components/healthcare/code/healthcare-bbe/ai-pair-programming.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: 'AI pair programming for your code'
-description: AI code gen like Github Copilot knows Ballerina. Ballerina knows healthcare. Why do all the work? Let AI do at least half of it for you! 
+description: AI-powered coding assistants like Github Copilot knows Ballerina. Ballerina knows healthcare. Why do all the work? Let AI do at least half of it for you! 
```

Comment:
```suggestion
description: AI-powered coding assistants like GitHub Copilot know Ballerina. Ballerina knows healthcare. Why do all the work? Let AI do at least half of it for you! 
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/9506/commits/114a2dd9512f0241a3961fc7d28b263351f0bc43